resources/definitions/cubicon_common.def.json

```diff
@@ -5,7 +5,8 @@
     "metadata": {
         "author": "Cubicon R&D Center",
         "manufacturer": "Cubicon",
-        "visible": false
+        "visible": false,
+        "has_materials": true
```

Comment:
This will require a version upgrade script! Without it, we can't accept this change.

Reply:
I will upgrade the version and commit it again.